Source/Core/Core/HW/DSPLLE/DSPDebugInterface.cpp

```diff
@@ -77,25 +81,21 @@ bool DSPDebugInterface::IsBreakpoint(unsigned int address)
 
 void DSPDebugInterface::SetBreakpoint(unsigned int address)
 {
-  int real_addr = DSPSymbols::Line2Addr(address);
+  int real_addr = Symbols::Line2Addr(address);
 
   if (real_addr >= 0)
   {
-    if (g_dsp_breakpoints.Add(real_addr))
-    {
-    }
+    g_dsp_breakpoints.Add(real_addr);
```

Comment:
I believe that empty `if` was added to silence a "return value not checked" warning somewhere; but I guess it doesn't matter anyways if we don't do anything in either case.